wolfcrypt/test/test.c

```diff
@@ -22721,9 +22721,9 @@ static int ecc_exp_imp_test(ecc_key* key)
 #else
     ecc_key    keyImp[1];
 #endif
-    byte       priv[32];
+    byte       priv[32*4];
```

Comment:
Why do we need to increase the size of these arrays?

Reply:
this was debug code at very early stage. Mistakenly left the code. Removed.